interop_client/src/main.rs

```diff
@@ -1124,6 +1124,11 @@ impl MlsClient for MlsClientImpl {
     ) -> Result<Response<ProposalResponse>, Status> {
         todo!()
     }
+    async fn free(&self, _request: Request<FreeRequest>) -> Result<Response<FreeResponse>, Status> {
```

Comment:
```suggestion

    async fn free(&self, _request: Request<FreeRequest>) -> Result<Response<FreeResponse>, Status> {
```

Could we add a newline?